src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/ContactTypeRepository.java

```diff
@@ -10,13 +10,11 @@
 public interface ContactTypeRepository extends JpaRepository<ContactType, Long> {
     Optional<ContactType> findByCode(String code);
 
-    /*
-        Category code "AL" refers to the "All" category.
-    */
-    @Query("SELECT DISTINCT type FROM ContactType type " +
-        "INNER JOIN FETCH type.contactCategories category " +
-        "WHERE category.codeValue = 'AL' AND type.selectable = true AND type.scheduleFutureAppointments = 'Y' " +
-        "AND type.attendanceContact = true")
+    @Query("""
```

Comment:
Filtering to the "All" category and those with scheduleFutureAppointments was unnecessarily restrictive, as those types are still available in Delius.